Statiq.Helpers/SocialImages.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Playwright;
+using Statiq.Common;
+using Statiq.Core;
+using Statiq.Html;
+using Statiq.Web;
+using Statiq.Web.Modules;
+using Statiq.Web.Pipelines;
+
+namespace Thirty25.Statiq.Helpers
+{
+    public class SocialImages : Pipeline
+    {
+        public SocialImages()
+        {
+            Dependencies.AddRange(nameof(Inputs), nameof(Data));
+
+            ProcessModules = new ModuleList
+            {
+                new GetPipelineDocuments(ContentType.Content),
+
+                // Filter to non-archive content
+                new FilterDocuments(Config.FromDocument(doc => !Archives.IsArchive(doc))),
+
+                // Process the content
+                new CacheDocuments
```

Comment:
I can see this part is copied from the `Content` pipeline, and that seems correct. I'll need to keep in mind once I get around to documenting the "create extra pipelines for Statiq Web" process that this is a good template. Your blog post will be a nice canonical example!